internal/installercache/installercache_test.go

```diff
@@ -161,42 +331,42 @@ var _ = Describe("installer cache", func() {
 		Expect(os.IsNotExist(err)).To(BeTrue())
 		_, err = os.Stat(r3)
 		Expect(os.IsNotExist(err)).To(BeFalse())
+		// Now measure disk usage, we should be under the cache size
+		Expect(getUsedBytesForDirectory(manager.config.CacheDir)).To(BeNumerically("<=", manager.config.MaxCapacity))
 	})
 
-	It("when cache limit is not set eviction is skipped", func() {
-		manager.storageCapacity = 0
+	It("evicts the oldest file", func() {
 		clusterId := strfmt.UUID(uuid.New().String())
-		r1, _ := testGet("4.8", "4.8.0", clusterId, false)
-		r2, _ := testGet("4.9", "4.9.0", clusterId, false)
-		r3, _ := testGet("4.10", "4.10.0", clusterId, false)
+		r1, l1 := testGet("4.8", "4.8.0", clusterId, false)
+		r2, l2 := testGet("4.9", "4.9.0", clusterId, false)
+		r3, l3 := testGet("4.10", "4.10.0", clusterId, false)
 
-		By("verify that the no file was deleted")
+		By("verify that the oldest file was deleted")
 		_, err := os.Stat(r1)
-		Expect(os.IsNotExist(err)).To(BeFalse())
+		Expect(os.IsNotExist(err)).To(BeTrue())
 		_, err = os.Stat(r2)
 		Expect(os.IsNotExist(err)).To(BeFalse())
 		_, err = os.Stat(r3)
 		Expect(os.IsNotExist(err)).To(BeFalse())
 
+		By("verify that the links were purged")
```

Comment:
Isn't this just testing that you called `Cleanup` in `testGet`?

Reply:
I guess it tests that cleanup actually cleans the links?
Code that was here before in any case, I tried not to interfere too much with existing tests

Reply:
The original reason was actually to test that the hardlink eviction was purging stuff. 

https://github.com/openshift/assisted-service/blob/master/internal/installercache/installercache_test.go#L71

So, with this in mind, I'll get rid of these verifications.